src/styles/themes/useThemePreference.ts

```diff
@@ -1,29 +1,34 @@
 import {useState, useEffect, useContext} from 'react';
-import {Appearance} from 'react-native';
+import {Appearance, ColorSchemeName} from 'react-native';
 import CONST from '../../CONST';
 import {PreferredThemeContext} from '../../components/OnyxProvider';
 
+// TODO: Remove this once "OnyxProvider" is typed
```

Comment:
Is there an issue for this? How could we track this without a TODO in code?

Reply:
Onyx is already typed now but i was expecting auto-complete/code suggestions working with it. Nevertheless, we can remove this TODO comment and type :)